package/tessel/tessel-version/files/version.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "0.0.1",
+  "commits": {
+    "openwrt": "8ec549f3d1a76647c24324a21c430c70cabf3b19",
```

Comment:
We don't particularly care about the commit hash of the openwrt repo, because it's referenced by this repo but changes less often.

Is the intention that this file is manually updated? Why not generate this file in a build script?